### ISSUE TITLE
Tokens Created Annonymously Don't Get Converted to Authenticated 

### ISSUE THREAD

**Issue by mkinnan** [signalpoint/push_notifications#16]
When I install the app on my Android phone, an anonymous token gets created the first time the app is loaded. When the user logs into the app, the token stays as anonymous. Should it be converted to an authenticated token or the anonymous be deleted and an authenticated token created?

**Comment by signalpoint** [comment]
@mkinnan I don't think so, the token is registered to the device itself and I don't think the system really cares about whether or not they are authenticated. I'd be curious if the tokens stored up in Drupal are tied to a user id in any way, if that was the case, then I could see value in that.
